node/src/Pos/Configuration.hs

```diff
@@ -97,6 +101,11 @@ networkDiameter = sec . ccNetworkDiameter $ nodeConfiguration
 networkConnectionTimeout :: HasNodeConfiguration => Microsecond
 networkConnectionTimeout = ms . fromIntegral . ccNetworkConnectionTimeout $ nodeConfiguration
 
+-- | Default is 30 seconds.
+conversationEstablishTimeout :: HasNodeConfiguration => Microsecond
```

Comment:
Oh, no, please, let's specify this explicitly in the configuration. Looking into the code is ~2 times psychologically harder than looking into config.yaml.

Reply:
Fine by me.